### PR TITLE
Upgrade ArgMin/ArgMax to opset3

### PR DESCRIPTION
**Pull request by kanvi-nervana**

### CODE REVIEW

File: ngraph_bridge/ngraph_mark_for_clustering.cc [hunk 1]
```diff
@@ -585,8 +585,8 @@ GetTFToNgOpMap() {
       {"AddV2", {std::make_shared<opset::Add>()}},
       {"Any", {std::make_shared<opset::ReduceLogicalOr>(), constant}},
       {"All", {std::make_shared<opset::ReduceLogicalAnd>(), constant}},
-      {"ArgMax", {std::make_shared<ngraph::op::ArgMax>()}},
-      {"ArgMin", {std::make_shared<ngraph::op::ArgMin>()}},
+      {"ArgMax", {std::make_shared<opset::Squeeze>(), constant}},
+      {"ArgMin", {std::make_shared<opset::Squeeze>(), constant}},
```
Comment (adk9) [inline ngraph_bridge/ngraph_mark_for_clustering.cc:589]: Add `opset::TopK` here.

Comment (kanvi-nervana) [reply ngraph_bridge/ngraph_mark_for_clustering.cc:589]: Yea I thought so too but NGRAPH_TF_LOG_PLACEMENT only shows Squeeze and constant.

File: ngraph_bridge/ngraph_builder.cc [hunk 2]
```diff
@@ -573,28 +563,53 @@ static Status TranslateArgMinMaxOp(
...
+  auto ng_k = ConstructNgNode<opset::Constant>(
+      op->name(), ng::element::i64, ng::Shape{}, std::vector<int>({1}));
+
+  std::string sort = "none";
+  auto ng_topk =
+      std::make_shared<opset::TopK>(ng_input, ng_k, k_axis, mode, sort, ng_et);
+  auto ng_indices = ng_topk->output(1);
+  int axis = ng_topk->get_axis();
+  auto axis_to_remove = ConstructNgNode<opset::Constant>(
+      op->name(), ng::element::i64, ng::Shape{}, std::vector<int>({axis}));
```
Comment (adk9) [inline ngraph_bridge/ngraph_builder.cc:593]: Use the same type here as the nGraph element type. Also, does aggregate initialization work here or do you have to use the `vector` constructor?

Finally, I think it'd be better to be explicit about the shape here.

**Comment by kanvi-nervana** [comment]
Depends on #652 for IE pipeline to pass

**Review by adk9 — APPROVED** [review]
LGTM.

File: test/python/tensorflow/tests_linux_ie_cpu.txt [hunk 3]
```diff
@@ -26,19 +26,22 @@ tests_common.txt
 [RUN]
 # Specify tests/patterns/regex that should be included
 
-argmax_op_test.ArgMaxTest.testFloatInt32Output
```
Comment (kanvi-nervana) [inline test/python/tensorflow/tests_linux_ie_cpu.txt]: Enable once #652 is merged

**Review by bani-intelaipg — APPROVED** [review]

**Pull request merged**